### DISCUSSION TITLE
getting build error on Cefsharp.Core

### DISCUSSION THREAD

**Discussion by ehsan1618** [cefsharp/CefSharp#4566] [General]
I cloned Cefsharp and trying to build it locally 
I am getting build errors on Cefsharp.Core external Dependencies on tuple file

Severity	Code	Description	Project	File	Line	Suppression State
Error	C2065	'tuple': undeclared identifier	CefSharp.Core	C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.32.31326\include\tuple	30	

and errors on following line 

template <bool _Same, class _Dest, class... _Srcs>
_INLINE_VAR constexpr bool _Tuple_conditional_explicit_v0 = false;

template <class... _Dests, class... _Srcs>
_INLINE_VAR constexpr bool _Tuple_conditional_explicit_v0<true, tuple<_Dests...>, _Srcs...> =
    !conjunction_v<is_convertible<_Srcs, _Dests>...>;

**Comment by amaitland** [comment]
You are likely missing the `VC++` build tools.

Continue the conversation at https://github.com/cefsharp/CefSharp/discussions/4567
